src/vm/dllimport.cpp

```diff
@@ -1142,9 +1142,180 @@ class ILStubState : public StubState
         }
         LOG((LF_STUBS, LL_INFO1000, "^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"));
 #endif // LOGGING
+
+#if !defined(PLATFORM_UNIX)
```

Comment:
What is preventing this from running on Unix?

Reply:
I think the ETW macros are only for Windows or do we generate appropriate stubs on Unix?

Reply:
Spoke to Vance about this and he believes that something 'appropriate' should be generated from the `.man` file on non-Windows. If this compiles, I would agree with @jkotas in that we should endeavor to enable this everywhere.

Reply:
Well, it builds with it all enabled. Thus, its probably perfect :) I'll fix the conditional logic.